systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PersistentVolumeClaimUtils.java

```diff
@@ -66,12 +67,24 @@ public static void waitUntilPVCAnnotationChange(String namespaceName, String clu
         LOGGER.info("PVC annotation has changed {}", newAnnotation.toString());
     }
 
-    public static void waitForPVCDeletion(String namespaceName, int volumesCount, JbodStorage jbodStorage, String clusterName) {
+    public static void waitForPvcDeletion(String namespaceName, String pvcName) {
+        TestUtils.waitFor("Wait for PVC deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Duration.ofMinutes(3).toMillis(), () -> {
+            if (kubeClient(namespaceName).getPvc(namespaceName, pvcName) == null) {
+                return true;
+            } else {
+                LOGGER.warn("PVC {} is not deleted yet! Triggering force delete by cmd client!", pvcName);
+                cmdKubeClient(namespaceName).deleteByName("pvc", pvcName);
+                return false;
+            }
+        });
+    }
+
+    public static void waitForJbodDeletion(String namespaceName, int volumesCount, JbodStorage jbodStorage, String clusterName) {
         int numberOfPVCWhichShouldBeDeleted = jbodStorage.getVolumes().stream().filter(
             singleVolumeStorage -> ((PersistentClaimStorage) singleVolumeStorage).isDeleteClaim()
         ).collect(Collectors.toList()).size();
 
-        TestUtils.waitFor("Wait for PVC deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Duration.ofMinutes(6).toMillis(), () -> {
+        TestUtils.waitFor("Wait for JBOD deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Duration.ofMinutes(6).toMillis(), () -> {
```

Comment:
```suggestion
        TestUtils.waitFor("Wait for JBOD storage deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Duration.ofMinutes(6).toMillis(), () -> {
```

mozna zmenit ja nazev te metody?